test/Serilog.Enrichers.Sensitive.Tests.Benchmark/PathMaskingOperatorBenchmarks.cs

```diff
@@ -0,0 +1,37 @@
+using System.Text.RegularExpressions;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Engines;
+
+namespace Serilog.Enrichers.Sensitive.Tests.Benchmark;
+
+[SimpleJob(RunStrategy.Throughput, warmupCount: 1)]
```

Comment:
The existing benchmark for regexes already proves a compiled regex is faster so adding another benchmark to prove it again isn't necessary. 
You can remove this class altogether.

Reply:
Class removed